src/VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -363,7 +337,7 @@ public void ConfigureServices(IServiceCollection services)
                          options.DiscoveryPath = Path.GetFullPath(options.DiscoveryPath ?? "modules");
                      })
                     .ValidateDataAnnotations();
-            services.AddModules(mvcBuilder);
+            services.AddModules(WebHostEnvironment.IsDevelopment(),x => mvcBuilder.AddApplicationPartWithRelatedAssembly(x));
```

Comment:
Yeah, seems it was the single reason why I couldn't move these extensions into the Module project.